cypress/integration/HomeworkTests/calculator.spec.js

```diff
@@ -0,0 +1,231 @@
+        homeworkPage.getNumberTwoField().type('2')
+        homeworkPage.getOperationDropdown().select('Multiply')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().contains('Number 1 is not a number')           
+    })
+})
+
+builds.forEach((build) =>{
+    it(`Should throw error message when trying to multiply letters in field two for build ${build}`, ()=>{
+        homeworkPage.getBuildDropdown().select(`${build}`)
+        homeworkPage.getNumberOneField().type('4')
+        homeworkPage.getNumberTwoField().type('2b')
+        homeworkPage.getOperationDropdown().select('Multiply')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().contains('Number 2 is not a number')           
+    })
+})
+
+//DIVIDE
+builds.forEach((build) =>{
+    it(`Should throw error message when trying to divide letters in field one for build ${build}`, ()=>{
+        homeworkPage.getBuildDropdown().select(`${build}`)
+        homeworkPage.getNumberOneField().type('4b')
+        homeworkPage.getNumberTwoField().type('2')
+        homeworkPage.getOperationDropdown().select('Divide')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().contains('Number 1 is not a number')           
+    })
+})
+
+builds.forEach((build) =>{
+    it(`Should throw error message when trying to divide letters in field two for build ${build}`, ()=>{
+        homeworkPage.getBuildDropdown().select(`${build}`)
+        homeworkPage.getNumberOneField().type('4')
+        homeworkPage.getNumberTwoField().type('2b')
+        homeworkPage.getOperationDropdown().select('Divide')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().contains('Number 2 is not a number')          
+    })
+})
+
+//CONCATENATE
+builds.forEach((build) =>{
+    it(`Should NOT throw error message when trying to concatenate letters in field one for build ${build}`, ()=>{
+        homeworkPage.getBuildDropdown().select(`${build}`)
+        homeworkPage.getNumberOneField().type('4b')
+        homeworkPage.getNumberTwoField().type('2')
+        homeworkPage.getOperationDropdown().select('Concatenate')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().should("not.be.visible")           
+    })
+})
+
+builds.forEach((build) =>{
+    it.only(`Should NOT throw error message when trying to concatenate letters in field two for build ${build}`, ()=>{
+        homeworkPage.getBuildDropdown().select(`${build}`)
+        homeworkPage.getNumberOneField().type('4')
+        homeworkPage.getNumberTwoField().type('2b')
+        homeworkPage.getOperationDropdown().select('Concatenate')
+        homeworkPage.getCalculateButton().click()
+        homeworkPage.getErrorField().should("not.be.visible")        
+    })
+})
+
+builds.forEach((build) =>{
+    it.only(`Should NOT let you select integers only checkbox two for build ${build}`, ()=>{
```

Comment:
The test description is incomplete. I would expect it sound like:
'Should NOT let you select integers only checkbox when Concatenate action is selected for ${build}'